boxwise_flask/resolvers.py

```diff
@@ -17,6 +17,12 @@ def resolve_all_camps(_, info):
     return list(response.dicts())
 
 
+@query.field("orgBases")
```

Comment:
how do you determine who has a right to see this info? is it considered public? common to anyone who works at any of the bases owned by the org? only accessable to people of a certain level at that org? we're going to start having to think about authorization concepts as we progress

Reply:
TBH I didn't think about it twice. Also I actually can't answer your questions who's allowed to see this and how the general authorization should be done. I just followed how "allBases" is being done and done the same